src/lib/nftClaimer/utils.ts

```diff
@@ -24,6 +24,7 @@ requiredEnvKeys.forEach(key => {
     missingEnvKeys.push(key);
   }
 });
+const broviderUrl = process.env.BROVIDER_URL;
```

Comment:
Could we maybe move the fallback value here, so the .env var is optional ?

Reply:
it's optional inside `snapshot.js`